crates/cairo-1-hint-processor/src/hint_processor.rs

```diff
@@ -121,6 +143,68 @@ impl Cairo1HintProcessor {
         .map_err(HintError::from)
     }
 
+    fn assert_le_find_small_arcs(
+        &self,
+        vm: &mut VirtualMachine,
+        exec_scopes: &mut ExecutionScopes,
+        range_check_ptr: &ResOperand,
+        a: &ResOperand,
+        b: &ResOperand,
+    ) -> Result<(), HintError> {
+        let a_val = res_operand_get_val(vm, a)?;
+        let b_val = res_operand_get_val(vm, b)?;
+        let mut lengths_and_indices = vec![
+            (a_val.clone(), 0),
+            (b_val.clone() - a_val, 1),
+            (Felt252::from(-1) - b_val, 2),
+        ];
+        lengths_and_indices.sort();
+        exec_scopes.assign_or_update_variable("excluded_arc", Box::new(lengths_and_indices[2].1));
+        // ceil((PRIME / 3) / 2 ** 128).
+        let prime_over_3_high = 3544607988759775765608368578435044694_u128;
+        // ceil((PRIME / 2) / 2 ** 128).
+        let prime_over_2_high = 5316911983139663648412552867652567041_u128;
+        let (range_check_base, range_check_offset) = extract_buffer(range_check_ptr);
+        let range_check_ptr = get_ptr(vm, range_check_base, &range_check_offset)?;
+        vm.insert_value(
+            range_check_ptr,
+            Felt252::from(lengths_and_indices[0].0.to_biguint() % prime_over_3_high),
+        )?;
+        vm.insert_value(
+            (range_check_ptr + 1)?,
+            Felt252::from(lengths_and_indices[0].0.to_biguint() / prime_over_3_high),
+        )?;
+        vm.insert_value(
+            (range_check_ptr + 2)?,
+            Felt252::from(lengths_and_indices[1].0.to_biguint() % prime_over_2_high),
+        )?;
+        vm.insert_value(
+            (range_check_ptr + 3)?,
+            Felt252::from(lengths_and_indices[1].0.to_biguint() / prime_over_2_high),
+        )
+        .map_err(HintError::from)
```

Comment:
shouldn't this use the `?` operator like the previous inserts?

Reply:
No need because it returns the Result